contracts/src/PBHEntryPointImplV1.sol

```diff
@@ -114,10 +127,23 @@ contract PBHEntryPointImplV1 is IPBHEntryPoint, WorldIDImpl, ReentrancyGuardTran
     /// @notice Thrown when the length of PBHPayloads on the aggregated signature is not equivalent to the amount of UserOperations.
     error InvalidAggregatedSignature(uint256 payloadsLength, uint256 userOpsLength);
 
+    /// @notice Thrown when the builder is not authorized to build blocks
+    error UnauthorizedBuilder();
+
+    /// @notice Thrown when there are no authorized builders
+    error InvalidAuthorizedBuilders();
```

Comment:
nit: Consider updating to `EmptyAuthorizedBuilders` for clarity as this is the only case where the `authorizedBuilders` array is invalid.